src/validation-error-message-formatter.js

```diff
@@ -318,13 +311,6 @@ exports.supportedExtensionsAttachmentReferenceViolation = function(itemPath, exp
   return 'attachment reference "' + itemPath + '" must have a supported file extension (' + extensionsString + ')';
 };
 
-/**
- * DEPRECATED. Use supportedExtensionsAttachmentReferenceViolation instead.
- */
-exports.supportedExtensionsAttachmentViolation = function(itemPath, expectedFileExtensions) {
```

Comment:
Since the validation-error-message-formatter module was moved to this location and the package has not been released since then, there is no good reason to keep this member around.